src/bin/scheduler/setup/directories.rs

```diff
@@ -56,18 +56,87 @@ pub fn setup(
     create_dir_all(&global_config.managed_directory)?;
 
     setup_results_directories(global_config, &plans, &ownership_setter)?;
+    #[cfg(windows)]
+    create_dir_all(&global_config.robocorp_home_base)?;
+
     let (surviving_plans, managed_dir_failures) = setup_managed_directories(plans);
+    #[cfg(windows)]
+    let (surviving_plans, robocorp_home_failures) =
+        setup_robocorp_home_directories(global_config, surviving_plans);
```

Comment:
We should only do this for plans that actually use RCC. I think this should become a step in `setup_rcc_working_directories`. In `setup_rcc_working_directories`, we can do something like
```
let mut rcc_plans_and_rcc_configs = Vec::new();
let mut system_plans = Vec::new();
for plan in plans.into_iter() {
    match plan.environment.clone() {
        Environment::Rcc(rcc_env) => {
            rcc_plans_and_env_build_dirs.push((plan, rcc_env.clone()))
        }
        _ => system_plans.push(plan),
    }
}
```
`rcc_plans_and_rcc_configs` can then be used as input for both `setup_environment_building_directories` and your new function. Ping me in case what I mean is unclear.

Reply:
I fully agree that we should not skip the system plans based on what we managed to setup as a robocorp home directory. That was a bug I missed.

However, I think the robocorp home should based on the `Session`, not `RCCEnvironment`. We are not setting up the home for each plan. I am open to an improvement, but this does not seem to be it to me.